src/App.js

```diff
@@ -1,12 +1,195 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import Counter from "./Components/Counter";
+
+
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+
+        const [count, setCount] = React.useState(
+            {aardbeien: 0,
+                bananen: 0,
+                appels:0,
+                kiwis:0
+            }
+        );
```

Comment:
Ik zou voor elk fruitsoort een aparte `useState` maken. Nu is het zo dat wanneer één van de fruitsoorten veranderd er een onnodige rerender plaatsvindt